tools/run_presubmit

```diff
@@ -69,20 +69,40 @@ def run_command(args: List[str],
 
 
 def get_merge_base() -> Optional[str]:
-  """Gets the merge base with the upstream branch (@{u}). Returns None if not
-  found."""
+  result = run_command(["git", "symbolic-ref", "--short", "HEAD"], check=True)
+  branch = result.stdout.strip()
+
+  # First check if there's a tracking branch
   try:
-    run_command(['git', 'rev-parse', '--abbrev-ref', '@{u}'],
-                check=True,
-                stderr=subprocess.DEVNULL)
-    result = run_command(['git', 'merge-base', 'HEAD', '@{u}'])
-    return result.stdout.strip()
+    # More direct way to get the upstream branch
```

Comment:
The nestead try/catch IMHO hurts readability.

WHy dion't we make run_command return none in case of failure? that's makes the code easier to read imho

Reply:
Would be nice but also it would mean changing this file signifcantly. I'd like to do this in a followup.